src/backend/gpopt/translate/CTranslatorQueryToDXL.cpp

```diff
@@ -4487,10 +4487,10 @@ CTranslatorQueryToDXL::CreateDXLProjectNullsForGroupingSets(
 
 		INT sort_group_ref = INT(target_entry->ressortgroupref);
 
-		GPOS_ASSERT_IMP(
-			nullptr == sort_grouping_col_mapping,
-			nullptr != sort_grouping_col_mapping->Find(&sort_group_ref) &&
-				"Grouping column with no mapping");
+		GPOS_ASSERT(nullptr != sort_grouping_col_mapping &&
+					nullptr !=
+						sort_grouping_col_mapping->Find(&sort_group_ref) &&
+					"Grouping column with no mapping");
```

Comment:
if below code calls AddSortingGroupingColumn function then sort_grouping_col_mapping->Find(&sort_group_ref) == nullptr and this assert raises before?

Reply:
```c
#define GPOS_ASSERT_IMP(x, y) GPOS_ASSERT(!(x) || (y))
```
so the equivalent expression in GPOS_ASSERT is
```
nullptr != sort_grouping_col_mapping || (nullptr != sort_grouping_col_mapping->Find(&sort_group_ref) &&
				"Grouping column with no mapping")
```